File: BlazorSalonApp/BlazorSalonApp/Dto/BusinessWithOwnerResponse.cs
namespace BlazorSalonApp.Dto;

public class BusinessWithOwnerResponse
{
    public long BusinessId { get; set; }
    public string BusinessName { get; set; } = string.Empty;
    public string BusinessSlug { get; set; } = string.Empty;
    public string BookingUrl { get; set; } = string.Empty;
    public string OwnerUsername { get; set; } = string.Empty;
    public string OwnerEmail { get; set; } = string.Empty;
    public string TemporaryPassword { get; set; } = string.Empty;
    public string Message { get; set; } = string.Empty;
}

File: Salon/src/main/java/com/example/salon/dto/BusinessWithOwnerResponse.java
package com.example.salon.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BusinessWithOwnerResponse {

    // Business info
    private Long businessId;
    private String businessName;
    private String businessSlug;
    private String bookingUrl;

    // Owner credentials (sendes til SYSTEM_ADMIN)
    private String ownerUsername;
    private String ownerEmail;
    private String temporaryPassword;

    // Message
    private String message;
}
